near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -345,7 +370,20 @@ impl NonFungibleToken {
         // Approval Management extension: return empty HashMap as part of Token
         let approved_account_ids =
             if self.approvals_by_id.is_some() { Some(HashMap::new()) } else { None };
+        
+        //construct the mint log as per the events standard
+        let nft_mint_log: EventLog = EventLog {
+            standard: NFT_STANDARD_NAME.to_string(), //standard name ("nep171")
+            version: NFT_METADATA_SPEC.to_string(), //version of the standard ("nft-1.0.0")
+            event: EventLogVariant::NftMint( vec![ NftMintLog { //the data related with the event stored in a vector
+                owner_id: token_owner_id.to_string(), //owner of the token
```

Comment:
Forgot to mention you'd also need to rename the following after getting rid of the clone:
```suggestion
                owner_id: owner_id.to_string(), //owner of the token
```